### PR TITLE
[KOGITO-5472] - Debug enabled scenarios are failing in kogito builder…

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
… image

Jira issue : https://issues.redhat.com/browse/KOGITO-5472

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: tests/features/kogito-common-builder-jvm.feature [hunk 1]
```diff
@@ -135,7 +135,7 @@ Feature: kogito-builder image JVM build tests
 #### SpringBoot Scenarios
 
   Scenario: Verify if the s2i build is finished as expected with debug enabled
-      Given s2i build https://github.com/kiegroup/kogito-examples.git from process-springboot-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
+      Given s2i build https://github.com/kiegroup/kogito-examples.git from `process-springboot-exampl`e using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```
Comment (radtriste) [inline tests/features/kogito-common-builder-jvm.feature]: I guess there was a small mistake done here ;)

```suggestion
      Given s2i build https://github.com/kiegroup/kogito-examples.git from process-springboot-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```

File: tests/features/kogito-common-builder-jvm.feature [hunk 2]
```diff
@@ -149,7 +149,7 @@ Feature: kogito-builder image JVM build tests
         | content_type         | application/json                                                              |
         | expected_status_code | 201                                                                           |
       And file /home/kogito/bin/process-springboot-example.jar should exist
-      And container log should contain main] .c.l.ClasspathLoggingApplicationListener
+      And container log should contain Started DemoApplication
```
Comment (radtriste) [inline tests/features/kogito-common-builder-jvm.feature:152]: Is that only shown in debug mode ?

**Review by radtriste — CHANGES_REQUESTED** [review]

**Review by ricardozanini — APPROVED** [review]

**Review by sutaakar — APPROVED** [review]
Looks good once Tristan's comments are addressed.

File: tests/features/kogito-common-builder-jvm.feature (hunk 2 above)
Comment (spolti) [reply tests/features/kogito-common-builder-jvm.feature:152]: I think these 3 checks are enough since the real check is done by the http request above.

**Review by spolti — APPROVED** [review]

File: tests/features/kogito-common-builder-jvm.feature (hunk 1 above)
Comment (vaibhavjainwiz) [reply tests/features/kogito-common-builder-jvm.feature]: done

**Comment by radtriste** [comment]
waiting for Jenkins to be back up to launch the missing main PR check

**Comment by radtriste** [comment]
Jenkins retest this

File: tests/features/kogito-common-builder-jvm.feature [hunk 3]
```diff
@@ -135,7 +135,7 @@ Feature: kogito-builder image JVM build tests
 #### SpringBoot Scenarios
 
   Scenario: Verify if the s2i build is finished as expected with debug enabled
-      Given s2i build https://github.com/kiegroup/kogito-examples.git from process-springboot-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
+      Given s2i build https://github.com/kiegroup/kogito-examples.git from `process-springboot-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```
Comment (radtriste) [inline tests/features/kogito-common-builder-jvm.feature]:
```suggestion
      Given s2i build https://github.com/kiegroup/kogito-examples.git from process-springboot-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```

**Comment by spolti** [comment]
I think it is ready now, @radtriste can we merge this one?

**Review by radtriste — APPROVED** [review]

**Pull request merged**